apps/studio/src/features/mail/service.ts

```diff
@@ -0,0 +1,40 @@
+import type { InvitationEmailTemplateData } from "./templates"
+import { createBaseLogger } from "~/lib/logger"
+import { isValidEmail } from "~/utils/email"
+import { sendMail } from "../../lib/mail"
+import { templates } from "./templates/templates"
+
+const logger = createBaseLogger({ path: "lib/mail/service" })
```

Comment:
i think this path might be `/features/mail/service`!

Reply:
nice catch, fixed in https://github.com/opengovsg/isomer/pull/1123/commits/8ca4497b3bf2d921d54866793dec3546966326a7 !